app/api/v1/routers/build_router.py

```diff
@@ -0,0 +1,83 @@
+from fastapi import Depends, HTTPException, APIRouter
+from sqlalchemy.orm import Session
+from uuid import UUID
+
+from app.db.database import get_db
+from app.db import crud
+# from app.schemas.build import _
+# To do: Set up schemas for builder
```

Comment:
I think schemas should be defined in the relevant layers. Seems like this is the the api layer so just place them in api/models.

Also use use "todo" instead of "to do" so IDEs can recognize them.